src/D3D/D3DDebugLogger.cpp

```diff
@@ -27,6 +27,7 @@ static std::array<const char*, 5u> messageSeverity{
 D3DDebugLogger::D3DDebugLogger(const Args& arguments) {
 
 #ifdef __ID3D12InfoQueue1_INTERFACE_DEFINED__
+	m_callBackCookie = 0;

```

Comment:
I am talking about this. This is copy assign, not constructing. Kind of a nitpick though.